app/scripts/platforms/extension.test.js

```diff
@@ -131,24 +131,5 @@ describe('extension platform', () => {
         `Transaction 1 failed! ${errorMessage}`,
       );
     });
-
-    it('should show failed transaction without nonce', async () => {
```

Comment:
The behavior this test was meant to capture was within an MMI code fence, so the test was no longer needed. See the diff of the code-under-test to see what I mean.